changebot/github/github_api.py

```diff
@@ -365,7 +432,7 @@ def last_commit_date(self):
             time = dateutil.parser.parse(date).timestamp()
             last_time = max(time, last_time)
         if last_time == 0:
-            raise Exception(f'No commit found in {url}')
+            raise Exception(f'No commit found in {self._url_commits}')
```

Comment:
Note: This is a real bug found during PEP8 check; `url` is undefined. Is the substitution correct?

Reply:
well, it would return ``f'{self._url_pull_request}/commits'``, so what about just simply  reporting ``self.repo`` here?

Reply:
Not sure. If you get to this line, then something was wrong in
```python
commits = paged_github_json_request(self._url_commits, headers=self._headers)
```
In that case, wouldn't you want to know the value of `self._url_commits`?

Reply:
I'm not sure, it depends how much useful info it has. Anyway, leave it for now, and if it's not verbose enough, we can always fix it later.

Reply:
Looks good